tendrl/commons/flows/__init__.py

```diff
@@ -250,8 +252,13 @@ def _execute_atom(self, atom_fqdn):
                     parameters=self.parameters
                 ).run()
                 return ret_val
-            except AtomExecutionFailedError as ex:
-                raise ex
+            except AtomExecutionFailedError:
```

Comment:
Already traceback is identified here https://github.com/Tendrl/commons/blob/master/tendrl/commons/jobs/__init__.py#L271

Reply:
The trace-back defined in https://github.com/Tendrl/commons/blob/master/tendrl/commons/jobs/__init__.py#L271 gives trace-back till this point and not any further files in import flow. When we add trace-back capture here, it goes till the error where any of the AtomExecutionFailedError occur. Currently we have a few of them after this PR, But still this gives a better traceback.